CMake debug build is broken

```
CMakeFiles/libqore.dir/lib/QoreNamespace.cpp.o: In function `StaticSystemNamespace::StaticSystemNamespace()':
/home/omusil/dev/qore/qore/lib/QoreNamespace.cpp:971: undefined reference to `init_debug_functions(QoreNamespace&)'
/usr/bin/ld: libqore.so.0.8.13: hidden symbol `_Z20init_debug_functionsR13QoreNamespace' isn't defined
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
CMakeFiles/libqore.dir/build.make:8942: recipe for target 'libqore.so.0.8.13' failed
make[2]: *** [libqore.so.0.8.13] Error 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -579,7 +579,7 @@ if (NOT HAVE_INET_PTON)
     set(LIBQORE_CPP_SRC ${LIBQORE_CPP_SRC} lib/inet_pton.cpp)
 endif ()
 if (${QORE_BUILD_TYPE_LWR} MATCHES "debug")
-    set(LIBQORE_QPP_SRC ${LIBQORE_QPP_SRC} lib/ql_debug.qpp)
+    set(LIBQORE_CPP_SRC ${LIBQORE_CPP_SRC} lib/ql_debug.cpp)
 endif ()
 
 # prepare libqore target

